integration_testing/features/coach/coach-edit-lesson-from-report.feature

```diff
@@ -0,0 +1,37 @@
+  # START testing this scenario with a FRESH DB (make a copy of the current if you want to reuse it later), and use the `kolibri manage importusers your-csv-file.csv` command to import a set of users for this case.
+
+Feature: Edit lesson details from its lesson report page
+  Class coaches and normal coaches are able to preview quizzes they create in the plan tab
+
+  Background:
+    Given I have finished creating a '<lesson>' Lesson A
+      And I am signed in to Kolibri as a <class coach> or <coach>
+      And the lesson can be inactive or active and assigned to anyone
+
+  Scenario: Edit lesson details from its report page
+    When as as a coach I go to Coach and click into the 'Reports' tab
+    Then I click into Lesson A report
```

Comment:
Combine the above When and Then into the "Given", so the starting point for the test is being on the Report page.

Reply:
Apply this to the other scenarios, so there is a shorter path to the Feature we're trying to test.